src/components/tl/ucp/bcast/bcast.c

```diff
@@ -7,6 +7,21 @@
 #include "tl_ucp.h"
 #include "bcast.h"
 
+ucc_base_coll_alg_info_t
+    ucc_tl_ucp_bcast_algs[UCC_TL_UCP_BCAST_ALG_LAST + 1] = {
+        [UCC_TL_UCP_BCAST_ALG_KNOMIAL] =
+            {.id   = UCC_TL_UCP_BCAST_ALG_KNOMIAL,
+             .name = "knomial",
+             .desc =
+                 "recursive k-ing with arbitrary radix (latency oriented alg)"},
```

Comment:
it is not "recursive k-ing"; it is "bcast over knomial tree with arbitrary radix"